boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/maven/liberty/LibertyPackageMojo.java

```diff
@@ -119,6 +119,10 @@ public void execute() throws MojoExecutionException {
 
             // Get booster dependencies from project
             Map<String, String> dependencies = MavenProjectUtil.getAllDependencies(project, BoostLogger.getInstance());
+            
+            // Determine the Java compiler target version and set this internally 
+            String javaCompilerTargetVersion = MavenProjectUtil.getJavaCompilerTargetVersion(project);
+            System.setProperty("boost.internal.compiler.target", javaCompilerTargetVersion);
```

Comment:
Also use the `BoostProperties` constant here.

Reply:
Done